daisy_workflows/image_import/import_image.sh

```diff
@@ -56,7 +56,12 @@ function resizeDisk() {
 }
 
 function copyImageToScratchDisk() {
-  local scratchDiskSizeGigabytes=${SOURCE_SIZE_GB}
+  # We allocate an extra 10% capacity to account for ext4's
+  # filesystem overhead. According to https://petermolnar.net/why-use-btrfs-for-media-storage/ ,
+  # the overhead is less than 10% using the default number of inodes
+  # and a reserved block percentage of 1%. We conserve a little more space
+  # by avoiding the reserved blocks via `mkfs.ext4 -m 0`.
+  local scratchDiskSizeGigabytes=$(awk "BEGIN {print int(${SOURCE_SIZE_GB} * 1.1)}")
```

Comment:
do we really need as much as 10%? If so, shall we handle the rounding issue? A couple of things to consider:
1. We can't create a disk less than 10GB. Shall we set 10GB as lower bound?
2. For example, a 18GB source image will cause a 19GB of scratch disk due to rounding, which is significantly less than 10%. Need to round up always.

Reply:
Based on https://petermolnar.net/why-use-btrfs-for-media-storage/ , 10% overhead is sufficient. Good point on rounding up -- I'll add that.

Reply:
As said in the last comment, 10GB is the lower bound of google cloud. Would you test what happens if we try create a <10GB disk?

Reply:
That's already handled here, right? Or were you thinking of something else?

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/dcd98d020cd82f0ef2edb9e90194b5fb08868589/daisy_workflows/image_import/import_image.sh#L73

Reply:
Cool! I missed it